Makefile

```diff
@@ -100,7 +100,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 
 .PHONY: deploy
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
-	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	#cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
```

Comment:
```suggestion
	cd config/deployment && $(KUSTOMIZE) edit set image quay.io/stolostron/hypershift-deployment-controller:latest=${IMG}
```

Reply:
And give the `IMG` a default value `quay.io/stolostron/`

Reply:
I'm addressing this in the new Basic Tests pull request. closing this pull request.
@zhujian7 I added the REPO URL you suggested and uncommented the line.